csrc/id_model/id_model_index_compute.h

```diff
@@ -62,7 +62,10 @@ class IdGraphIndexCompute : public OptOutDispatch {
   }
 
   void setIndex(IterDomain* id, Val* idx) {
-    index_map_.emplace(toGroup(id), idx);
+    // May overwrite index. When the graph is cyclic due to, e.g.,
+    // resize, the index obtained by traversing most through the
```

Comment:
"by traversing _most recently_"? So the index map will reflect whichever Expr was most recently processed?

Reply:
Yes. It's a good question as this will have some related change in a follow-up PR 😃